README.md

```diff
@@ -60,16 +60,29 @@ Apostrophe core piece types and pages are prefixed `AposCore`. For example, incl
 
 ### Rendered content
 
-Apostrophe pieces are well suited to being delivered as structured, JSON-like data. Apostrophe pages, on the other hand, usually consist primarily of the flexible content "areas," populated with any number of different widget types. It powers Apostrophe's great in-context editing experience, but the raw data is not easy to use in a page template on your own.
+Apostrophe "pieces" are well suited to being delivered as structured, JSON-like data. However, the power of Apostrophe really shines when editors build custom series of content widgets in "areas." Because areas can contain many types of widgets, and thus are not consisten in data structure, it is more useful to retrieve those from the APIs as rendered HTML.
 
-Unless you add `renderPages: false` to this source plugin's options, pages will appear with a `_rendered` property in Gatsby GraphQL queries. This property's value is a string of HTML, rendered using the relevant page template in the Apostrophe app. That HTML can be used in your Gatsby site to [programmatically create pages](https://www.gatsbyjs.com/docs/tutorial/part-seven/) using the right layout component and slug structure for your site.
+Area fields on piece types will be available in the GraphQL queries with a `_rendered` property containing a string of rendered HTML. All other field types are delivered directly as their normal data types, e.g., strings, number, arrays, etc.
+
+#### Rendered pages
+
+Apostrophe pages usually consist primarily of these content "areas," populated with any number of different widget types. Unless you add `renderPages: false` to this source plugin's options, pages will appear with a `_rendered` property in Gatsby GraphQL queries. This property's value is a string of HTML, rendered using the relevant page template in the Apostrophe app. See [the page API documentation](https://a3.docs.apos.dev/reference/api/pages.html#get-url-apos-refresh-1) for more specifics on what is rendered.
```

Comment:
Really think you ought to be able to do this for pieces too, as they often have show pages, although it doesn't have to be the default. Just feels incomplete not to offer that. Some pieces are also "primarily consisting of content areas."

But, that can wait for an update. Not a blocker.

Reply:
OK, it sounds like the responses for pages and pieces are fundamentally different, the page renders a whole, while piece areas render individually. Nnngh. It bugs me that we don't have the option of doing either for either case. But OK.

Reply:
Oh WAIT a minute, you CAN do this for pieces. It's just apos-refresh plus the new query parameter which is just a convention. Confusing. May as well mention you can do this for anything, page or piece, that comes back from the original API request with a `_url` property.

Reply:
The plugin is doing this stuff for people, so they never need to know about `apos-refresh` or any other query parameters. We can give people options to enable this or that functionality, but this is not where we should tell people how to make headless requests. The link to the docs for `apos-refresh` is for the same of not repeating what template material is included.

We can add functionality later for fully rendered piece pages, but 1) there are good reasons I don't think we should do this that we can discuss later and 2) this implementation follows [the technical design](https://apostrophecms.atlassian.net/browse/GPA3-5) we discussed previously, which described delivering pieces as the data objects with areas rendered as HTML.